src/pages/well-architected-framework/[...tutorialSlug]/index.tsx

```diff
@@ -0,0 +1,175 @@
+import { serializeContent } from 'views/tutorial-view/utils/serialize-content'
+import { getTutorial } from 'lib/learn-client/api/tutorial'
+import { getCollectionsBySection } from 'lib/learn-client/api/collection'
+import { stripUndefinedProperties } from 'lib/strip-undefined-props'
+import { splitProductFromFilename } from 'views/tutorial-view/utils'
+import { getCollectionContext } from 'views/tutorial-view/utils/get-collection-context'
+import wafData from 'data/well-architected-framework.json'
+import wafContent from 'content/well-architected-framework/index.json'
+import { buildCategorizedWafSidebar } from 'views/well-architected-framework/utils/generate-sidebar-items'
+import WellArchitectedFrameworkTutorialView from 'views/well-architected-framework/tutorial-view'
+import {
+	Collection as ApiCollection,
+	TutorialLite as ApiTutorialLite,
+} from 'lib/learn-client/types'
+import { generateTopLevelSidebarNavData } from 'components/sidebar/helpers'
+import { MenuItem, SidebarProps } from 'components/sidebar'
+import { generateWafCollectionSidebar } from 'views/well-architected-framework/utils/generate-collection-sidebar'
+import { getNextPrevious } from 'views/tutorial-view/components'
+
+export async function getStaticProps({ params }) {
+	const [collectionFilename, tutorialFilename] = params.tutorialSlug
+	const currentPath = `/${wafData.slug}/${params.tutorialSlug.join('/')}`
+
+	// get all the waf collections to generate the collection level sidebar
+	const allWafCollections = await getCollectionsBySection(wafData.slug)
+	const currentCollection = allWafCollections.find(
+		(collection) => collection.slug === `${wafData.slug}/${collectionFilename}`
+	)
+	const currentTutorialReference = currentCollection?.tutorials.find((t) =>
+		t.slug.endsWith(tutorialFilename)
+	)
+
+	// The tutorial doesn't exist in collection - return 404
+	if (!currentCollection || !currentTutorialReference) {
+		return null
+	}
+
+	// Need to get tutorial data with full mdx content
+	const fullTutorialData = await getTutorial(currentTutorialReference.slug)
```

Comment:
There's a _little_ bit of duplication between here and the product tutorial prop gen. But its different enough to make me think this is fine 👌